tests/test_main.cpp

```diff
@@ -281,6 +281,10 @@ CATCH_REGISTER_LISTENER( CataListener )
 
 int main( int argc, const char *argv[] )
 {
+#ifdef _WIN32
+    int *p = NULL;
+    *p = 0;
+#endif
```

Comment:
The compiler may just optimize this obvious null dereference away (It's undefined behavior, not guaranteed crash). `std::abort` can be used instead to crash the program.